LIBVmonitor0.11/LIBVmonitor0.11.ino

```diff
@@ -162,31 +194,27 @@ void loop() {
 }
 
 
-void alarmTrigger(bool _al_stat, int _al_num){
-	int _num_count[3];
-	long _time_on[3];
-	long _time_of[3];
-  	if(_al_stat && (al_count<=_num_count[_al_num]){
-  		if(!buzzer_stat && (millis()-al_t >= _time_on[_al_num])){
-  			al_t = millis();
-  			al_stat = true;
-  			al_count++;
-  		}else if(buzzer_stat && (millis()-al_t >= _time_of[_al_num])){
-  			al_t = millis();
-  			al_stat = false;
-  		}
-  	}else al_count=0;
-}
-
 void BEEP(){
   digitalWrite(BUZZER, 1);
-  delay(200);
+  delay(100);
   digitalWrite(BUZZER, 0);
 }
 
+void EEPROM_update(){
+	EEPROM.update(TV_ADD, TV/10);
+    EEPROM.update(TI_ADD, TI/100);
+    EEPROM.update(IE_ADD, IE);
+}
+
+void EEPROM_read(){
+	TV = EEPROM.read(TV_ADD)*10;
+  	TI = EEPROM.read(TI_ADD)*100;
+  	IE = EEPROM.read(IE_ADD);
+}
+
 
-void wireData(){
-  Wire.beginTransmission(8);
+void wireData(uint8_t _add){
+  Wire.beginTransmission(_add);
   Wire.write(TV/10);
```

Comment:
Why are we dividing by 10 and 100 on these? As before, change them to #defines. Or, if it's a simple conversion, add a comment here explaining.